packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -87,25 +87,6 @@ export interface PortMapping {
   readonly protocol: Protocol;
 }
 
-/**
- * Represents the properties needed to define healthy and active listeners for nodes
- */
-export interface VirtualNodeListener {
-  /**
-   * Array of PortMappingProps for the listener
-   *
-   * @default - HTTP port 8080
-   */
-  readonly portMapping?: PortMapping;
-
-  /**
-   * Health checking strategy upstream nodes should use when communicating with the listener
-   *
-   * @default - no healthcheck
-   */
-  readonly healthCheck?: HealthCheck;
-}
-
```

Comment:
I believe that when https://github.com/aws/aws-cdk/pull/11277 is merged, you can update this PR on top of it, and then remove the `PortMapping` interface completely.